src/Components/SoundUploader/index.js

```diff
@@ -0,0 +1,33 @@
+import React, { useRef, useState } from "react";
+import styled from "styled-components";
+
+const FileInput = styled.input``;
+
+const SoundUploader = ({}) => {
+  const inputEl = useRef(null);
+
+  const uploadSound = event => {
+    const file = event.target.files[0];
```

Comment:
before calling `event.target.files[0]` it would be good to put some error checking in place. You have two options, you can either do something like:
```
try {
const file = event.target.files[0];
....
} catch (error) {
     alert("There was an error processing your file");
}
```
or 
```
if (event && event.target && event.target.files && event.target.files[0]) {
...
}
```

- [ ] Add error checking

Reply:
try/catch block has been added to updated version of branch. thank you for catching this my bad on forgetting to put in error checks!